common/src/main/java/org/terracotta/angela/common/distribution/Distribution102Controller.java

```diff
@@ -363,6 +365,16 @@ public void stopTms(File installLocation, TerracottaManagementServerInstanceProc
     }
   }
 
+  @Override
```

Comment:
While we're here, we should add voter support in `Distribution102Controller` as well. @aurbroszniowski needed this change to write smoke tests, for which he must have done the changes here. @mobasherul can you please sync-up with Aurelien, get his changes, merge his and yours, and update this PR?